app/middlewares/session_validation.js

```diff
@@ -0,0 +1,24 @@
+const errors = require('../errors');
+const jwt = require('jwt-simple');
+const {
+  common: {
+    session: { secret }
+  }
+} = require('../../config');
+const { findUser } = require('../services/users');
+
+exports.checkSession = (req, _, next) => {
```

Comment:
why the _?

Reply:
I've seen is a good practice to replace parameters that are unused with underscore

Reply:
ok, but keep in mind we dont use to do that in an endpoint